main.cpp

```diff
@@ -42,24 +42,531 @@ Purpose:  This project continues developing Project3.
+{
+    std::cout << "Initialized Value of storeName: " << storeName << std::endl;
+    std::cout << "double PizzaStore::billCustomer(std::string, int, float) " << pizzaType << std::endl;
+    double bill;
+    if (pizzaType == "regular")
+        discountSale = .20f;
+    bill = (20 + (.5 * toppings)) * static_cast<double>(discountSale); 
+    return bill;
+}
+
+int PizzaStore::addToppings(std::string toppingName, bool maxedOutToppings)
+{
+    std::cout << "Initialized Value of numOvens: " << std::endl;
+    std::cout << "int PizzaStore::addToppings(std::string, bool) " << toppingName << std:: endl;
+    if (maxedOutToppings == false && toppingName != "Pineapple")
+        numToppings += 1;
+    return numToppings;
+}
+
+bool PizzaStore::hireOrFireEmployee(std::string employeeName, int totalStaffMembers)
+{
+    std::cout << "Initialized Value of totalMonthlySales: " << totalMonthlySales << std::endl;
+    std::cout << "bool PizzaStore::hireOrFireEmployee(std::string, int) " << employeeName << std::endl;
+    bool hire;
+    if (employeeName != "None" || totalStaffMembers < 25)
+    {
+        numEmployees += 1;
+        hire = true;
+    }
+    else
+    {
+        numEmployees -= 1;
+        hire = false;
+    }
+    return hire;
+}
+
+double PizzaStore::getPizzaStore(int threshold, int startingValue)
+{
+    PizzaStore ps1;
+    while (startingValue <= threshold)
+    {
+        ps1.numToppings += (startingValue + 4);
+        ps1.totalMonthlySales += 100.25;
+        if (ps1.numToppings > threshold)
+            break;
+    }
+    return ps1.totalMonthlySales;
+}
+
 /*
  new UDT 4:
  with 2 member functions
  */
 
+struct Divisions
+{
+    // member variables
+    Gym gym_north;
+    Gym gym_south;
+    Gym gym_east;
+    Gym gym_west;
+
+    // member functions
+    int multiplyTotalNumberUsers(int multipler);
+    double saveCostOfElectricity();
+
+    // Constructor
+    Divisions();
+
+    // Destructor
+    ~Divisions();
+
+};
+
+Divisions::Divisions()
+{
+    std::cout << "Creating an object of the Divisions structure." << std::endl;
+    gym_north.numUsers = gym_north.findGymLocation(50, 0);
+    gym_south.numUsers = gym_south.findGymLocation(75, 0);
+    gym_east.numUsers = gym_east.findGymLocation(100, 0);
+    gym_west.numUsers = gym_west.findGymLocation(125, 0);
+}
+
+Divisions::~Divisions()
+{
+    std::cout << "Division object is getting destroyed. " << std::endl;
+}
+
+int Divisions::multiplyTotalNumberUsers(int multipler)
+{
+    std::cout << "Multiplying all divisions by " << multipler << std::endl; 
+
+    return multipler * (gym_north.numUsers + gym_south.numUsers + gym_east.numUsers + gym_west.numUsers);
+}
+
+double Divisions::saveCostOfElectricity()
+{
+    double tempCost;
+    
+    tempCost = gym_north.costOfMonthlyElectricity + gym_south.costOfMonthlyElectricity + gym_east.costOfMonthlyElectricity + gym_west.costOfMonthlyElectricity;
```

Comment:
`double tempCost = `
initialize your variables when you declare them
- [ ] revise